test/functional/legacy-fixtures/regression/legacy-gh-16/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,15 @@
+'@fixture GH-16';
+'@page ./index.html';
+
+
+'@test'['Check stored data'] = {
+    '1.Set data and reload iframe': inIFrame('#iframe', function () {
+        this.data = 200;
+
+        act.click('#replaceSrc');
+    }),
+
+    '2.Check data': function () {
+        ok(this.data.toString(), '200');
```

Comment:
but the issue is reproduced on my side when I just use `ok(this.data, 200);`. So, maybe it was some another reason of this issue?